src/scala/com/github/johnynek/bazel_deps/CreatePom.scala

```diff
@@ -7,7 +7,9 @@ object CreatePom {
     def toXml: Elem = {
       <dependency>
         <groupId>{self.group.asString}</groupId>
-        <artifactId>{self.artifact.asString}</artifactId>
+        <artifactId>{self.artifact.asString.split(":")(0)}</artifactId>
```

Comment:
why is this split needed? why isn't it just `self.artifact.artifactId`?

Reply:
Since we're concatenating the packaging and classifier in [MavenArtifactId](https://github.com/bazeltools/bazel-deps/blob/b5c72316251af563b07516d835b981737836bc5c/src/scala/com/github/johnynek/bazel_deps/DepsModel.scala#L465) this pulls out just the ArtfiactId string. In the current state the MavenArtifactId.asString is really the artifact id, classifier, and packaging all concatenated together.

Reply:
but look here:
https://github.com/bazeltools/bazel-deps/blob/b5c72316251af563b07516d835b981737836bc5c/src/scala/com/github/johnynek/bazel_deps/DepsModel.scala#L459

isn't `self.artifact.asString.split(":")(0) == self.artifact.artifactId`?

I understand the concatenation, but I'm saying access it directly *before* the concatenation. Note the confusing artifact vs artifactId difference. I'm not saying `self.artifact.asString` I'm saying `self.artifact.artifactId`.

Reply:
Oh gotcha yeah that makes sense! will update that's much clearer